internal/native/message_server.go

```diff
@@ -1,99 +1,7 @@
-bool pactffi_cleanup_mock_server(int mock_server_port);
-char* pactffi_mock_server_mismatches(int mock_server_port);
-bool pactffi_mock_server_matched(int mock_server_port);
-
-// Functions to get message contents
-
-// Get the length of the request contents of a `SynchronousMessage`.
-size_t pactffi_sync_message_get_request_contents_length(SynchronousMessage *message);
-struct PactSyncMessageIterator *pactffi_pact_handle_get_sync_message_iter(PactHandle pact);
-struct SynchronousMessage *pactffi_pact_sync_message_iter_next(struct PactSyncMessageIterator *iter);
-
-// Async
-// Get the length of the contents of a `Message`.
-size_t pactffi_message_get_contents_length(Message *message);
-
-//  Get the contents of a `Message` as a pointer to an array of bytes.
-const unsigned char *pactffi_message_get_contents_bin(const Message *message);
-struct PactMessageIterator *pactffi_pact_handle_get_message_iter(PactHandle pact);
-struct Message *pactffi_pact_message_iter_next(struct PactMessageIterator *iter);
-
-// Need the index of the body to get
-const unsigned char *pactffi_sync_message_get_response_contents_bin(const struct SynchronousMessage *message, size_t index);
-size_t pactffi_sync_message_get_response_contents_length(const struct SynchronousMessage *message, size_t index);
-
-// Sync
-// Get the request contents of a `SynchronousMessage` as a pointer to an array of bytes.
-// The number of bytes in the buffer will be returned by `pactffi_sync_message_get_request_contents_length`.
-const unsigned char *pactffi_sync_message_get_request_contents_bin(SynchronousMessage *message);
-// Set Sync message request body - non binary
-void pactffi_sync_message_set_request_contents(InteractionHandle *message, const char *contents, const char *content_type);
-
-// Set Sync message request body - binary
-void pactffi_sync_message_set_request_contents_bin(InteractionHandle *message, const unsigned char *contents, size_t len, const char *content_type);
-
-// Set sync message response contents - non binary
-void pactffi_sync_message_set_response_contents(InteractionHandle *message, size_t index, const char *contents, const char *content_type);
-
-// Set sync message response contents - binary
-void pactffi_sync_message_set_response_contents_bin(InteractionHandle *message, size_t index, const unsigned char *contents, size_t len, const char *content_type);
-
-// Can be used instead of the above as a general abstraction for non-binary bodies
-bool pactffi_with_body(InteractionHandle interaction, int interaction_part, const char *content_type, const char *body);
-
-// Can be used instead of the above as a general abstraction for binary bodies
-// bool pactffi_with_binary_file(InteractionHandle interaction, int interaction_part, const char *content_type, const uint8_t *body, size_t size);
-bool pactffi_with_binary_file(InteractionHandle interaction, int interaction_part, const char *content_type, const char *body, int size);
+#include "pact.h"
```

Comment:
I swear I tried this previously and it didn't work. Thanks - probably was an unrelated issue when I was first dealing with all of this!